### DISCUSSION TITLE
Example with custom text markers?

### DISCUSSION THREAD

**Discussion by jaanaltosaar** [observablehq/plot#2197] [Q&A]
Hi! I am generating a visualization that involves Mandarin and need to have markers such as `你` or `好` in an Observable Framework plot. 

Would this be possible? Are there examples of custom markers/labels?

**Comment by Fil** [comment]
Please open questions in the [discussion](https://github.com/observablehq/framework/discussions/categories/q-a) section. This is for issues (I'll migrate it to a discussion).

**Comment by Fil** [comment]
The **[text](https://observablehq.com/plot/marks/text)** mark supports the whole range of unicode. For annotations, you sometimes want to anchor them to a given side of the frame (left, right, top-left etc), instead of to a data point. In that case you use **frameAnchor**.

If you only want to display a single piece of text, pass it as an array:

```js
Plot.text(["✅"], { frameAnchor: "top-right" }).plot({ margin: 10 })
```
